src/main/resources/story/web/createBoardAndCards.story

```diff
@@ -0,0 +1,23 @@
+Scenario: Log-in
+When I login with ${email} and ${password}
+When I click on element located `id(login-submit)`
+
+Scenario: Create a new board
+When I wait until element located `By.xpath(//*[@data-testid="create-board-tile"])` appears
+When I click on element located `By.xpath(//*[@data-testid="create-board-tile"])`
+When I wait until element located `By.xpath(//*[@data-testid="create-board-title-input"])` appears
+When I enter `#{generate(regexify '[a-z]{5}[A-Z]{2}')}` in field located `By.xpath(//*[@data-testid="create-board-title-input"])`
+When I click on element located `By.xpath(//*[@data-testid="create-board-submit-button"])`
+When I wait until element located `By.xpath(//*[@data-testid="filter-popover-button"])` appears
+Then number of elements found by `By.xpath(//*[@data-testid="filter-popover-button"])` is = `1`
```

Comment:
It would be better to verify that correct board was created - please add verification that the board with expected attributes (name) is displayed

Reply:
Last report (20/01/2023)
[allure.zip](https://github.com/Yanina-Matuzava/test-project-matuzava/files/10466653/allure.zip)